src/test/java/seedu/address/logic/parser/AddressBookParserTest.java

```diff
@@ -104,11 +99,28 @@ public void parseCommand_addInsuranceCommand() throws Exception {
                 + " "
                 + PREFIX_INSURANCE_ID
                 + " 0 ";
-        System.out.println(userInput);
         AddInsuranceCommand command = (AddInsuranceCommand) parser.parseCommand(userInput);
         assertEquals(new AddInsuranceCommand(INDEX_FIRST_PERSON, 0), command);
     }
 
+    @Test
+    public void parseCommand_deleteInsuranceCommand() throws Exception {
+        String userInput = DeleteInsuranceCommand.COMMAND_WORD
+                + " "
+                + INDEX_FIRST_PERSON.getOneBased()
+                + " "
+                + PREFIX_INSURANCE_ID
+                + " 0 ";
+        DeleteInsuranceCommand command = (DeleteInsuranceCommand) parser.parseCommand(userInput);
+        assertEquals(new DeleteInsuranceCommand(INDEX_FIRST_PERSON, 0), command);
+    }
+
+    @Test
+    public void parseCommand_unrecognisedInput_throwsParseException() {
```

Comment:
re-ordered to move add and delete insurance above the tests for unrecognised inputs

Reply:
Sure 👍